packages/stylable/src/selector-utils.ts

```diff
@@ -260,3 +260,41 @@ export function isChildOfAtRule(rule: postcss.Rule, atRuleName: string) {
 export function isCompRoot(name: string) {
     return name.charAt(0).match(/[A-Z]/);
 }
+
+export function createWarningRule(
+    extendedNode: string,
+    scopedExtendedNode: string,
+    extendingNode: string,
+    scopedExtendingNode: string) {
+    // tslint:disable-next-line:max-line-length
+    const message = `"Invalid CSS class assignment of '.${scopedExtendingNode}', target node is missing extended class '.${scopedExtendedNode}'"`;
+    return postcss.rule({
+        selector: `.${extendingNode}:not(.${extendedNode})::before`,
+        nodes: [
+            postcss.decl({
+                prop: 'content',
+                value: message
+            }),
+            postcss.decl({
+                prop: 'width',
+                value: `200px !important`
+            }),
+            postcss.decl({
+                prop: 'height',
+                value: `200px !important`
```

Comment:
200x200?

Reply:
I need to look at how it shows on the screen, but I had a small floating message in mind, not a 200x200 block. Ideally, like @barak007 said, this should be coupled with dev run-time code that will move this message to the console.